hawc/apps/assessment/models.py

```diff
@@ -746,6 +747,8 @@ def get_dict(self):
         }
 
     def get_assessment(self):
+        if self.content_object is None:
```

Comment:
It may be better to do `hasattr(self.content_object,"get_assessment")`

Reply:
I tried that! It's failing when in some cases self.content_object is None (if the object is deleted I think). I think this way may be better b/c it gives us more data that we can't retrieve a `content_object`